db/migrate/20170720142145_add_deb_release_parameters_to_repositories.rb

```diff
@@ -0,0 +1,7 @@
+class AddDebReleaseParametersToRepositories < ActiveRecord::Migration
+  def change
+    add_column :katello_repositories, :deb_releases, :string, limit: 255
+    add_column :katello_repositories, :deb_components, :string, limit: 255
+    add_column :katello_repositories, :deb_architectures, :string, limit: 255
```

Comment:
Debian (apt) repositories have different structure than yum.
They can contain packages for severals architectures, be divided into multiple components (think: main, contrib, non-free) and incorporate more than one Release.
That is why i introduced these new fields as plural terms.